src/main/java/core/basesyntax/model/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+import core.basesyntax.suppliers.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    private static FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(10)];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+        }
+        for (Figure figure : figures) {
+            figure.draw();
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/185674801-76eed132-e5c9-4b89-8802-74d7d89ba37e.png)
The task was not correctly implemented
You should get half of figures random, another half should be default